Makefile

```diff
@@ -100,4 +100,8 @@ dist/binary.md:
 		printf "[$$ff](https://github.com/codenotary/immugw/releases/download/v${VERSION}/$$ff) | $$shm_id \n" ; \
 	done
 
+.PHONY: dist
+dist: dist/binaries
```

Comment:
This is to keep it consistent with immudb Makefile ?

Reply:
ack, just wanted to follow the same convention